app/services/default_shipping_category.rb

```diff
@@ -0,0 +1,13 @@
+# Encapsulates the concept of default stock location in creation of a product or a shipping method.
+
+class DefaultShippingCategory
+  NAME = 'Default'.freeze
```

Comment:
OFN is deployed in different places around the world, and translations are quite important. For example, there might be some languages where users might not understand what "Default" means.

I checked all [managed instances](https://github.com/openfoodfoundation/ofn-install/wiki/Current-deployment-status). and all still have the "Default" (English) shipping category. But if for example OFN France updates their "Default" to "Défaut", the "Default" category would automatically be created again so they would end up with 2.

This is not an immediate issue that should be fixed, I think. I'll create a separate issue for this.

Reply:
This is a good point; I'll keep in mind that any user-facing text should be set up with a key in en.yml so Transifex can build out the translation. If you are setting up an issue for this, I modeled `default_shipping_category.rb` after `default_stock_location.rb`, which will need the same fix.

Reply:
Thanks, @daningenthron!

I created issue #4360 for making the "Default" shipping category translatable.

I didn't create an issue for stock location anymore because we only support one stock location, so this is mostly used internally. :smiley: